test/integration/good/function-signatures/distributions/univariate/discrete/hypergeometric/pretty.expected

```diff
@@ -17,3 +17,9 @@ model {
   y_p ~ normal(0, 1);
 }
 
+Warning in 'hypergeometric_log.stan', line 6, column 26: hypergeometric_log
+    is deprecated and will be removed in Stan 2.32.0. Use hypergeometric_lpmf
+    instead. This can be automatically changed using stanc --print-canonical
```

Comment:
Just to check, have you verified that the canonicalizer works properly on these?

Otherwise this looks great

Reply:
No, I didn't test the canonicalizer at all. But the canonicalizer gets the function renames directly from `Deprecation_analysis.ml` so `--print-canonical` will do exactly as the warning says.

Maybe we should have a test that runs the canonicalizer on every warning test file and checks that the warnings are gone.

Reply:
That wouldn't be a terrible idea, but it would be difficult to instrument (and I agree with Rok in #1049 that we should try to have all the tests which aren't intentionally testing this up to date). For now it might be worth just adding a few of these new signatures to `test/integration/canonicalize/deprecated.stan`